internal/controller/services/gateway/resources/kube-auth-proxy-oidc-deployment.tmpl.yaml

```diff
@@ -14,6 +15,7 @@ spec:
     metadata:
       labels:
         app: {{.KubeAuthProxyServiceName}}
+        {{.ComponentLabelKey}}: {{.ComponentLabelValue}}
       annotations:
```

Comment:
internal/controller/services/gateway/resources/kube-auth-proxy-oauth-deployment.tmpl.yaml and internal/controller/services/gateway/resources/kube-auth-proxy-oidc-deployment.tmpl.yaml looks same